README.md

```diff
@@ -1,40 +1,20 @@
 # Icinga Notifications
 
-> **Warning**
-> This is an early preview version for you to try, but do not use this in production. There may still be severe bugs
-> and incompatible changes may happen without any notice. At the moment, we don't yet provide any support for this.
+> [!WARNING]
+> This is an early preview version for you to try, but do not use this in production.
+> There may still be severe bugs and incompatible changes may happen without any notice.
+> At the moment, we don't yet provide any support for this.
 
 Icinga Notifications is a set of components that processes received events from various sources, manages incidents and
 forwards notifications to predefined contacts, consisting of:
 
-* The Icinga Notifications daemon (this repository), which receives events and sends notifications
-* An [Icinga Web module](https://github.com/Icinga/icinga-notifications-web) that provides graphical configuration and further processing of the data collected by the daemon
-* And Icinga 2 and other custom sources that propagate state updates and acknowledgement events to the daemon
+* The Icinga Notifications daemon (this repository), which receives events and sends notifications.
+* The [Icinga Notifications Web](https://github.com/Icinga/icinga-notifications-web) module,
+  which provides graphical configuration.
+* Icinga 2 or other sources that provide monitoring events that result in notifications.
```

Comment:
Icinga 2 is IMHO a must have, so `s/or/and/`. The same applies to the about page.

Reply:
Is it technically really a hard requirement? I mean, you might end in a hard place without a real host and service source..

But yes, valid point. It's all about Icinga anyways.

Reply:
That's what I meant in our discussion the other day. The daemon itself is essentially "standalone" while still being branded with the Icinga ecosystem.
No, there's no hard-dependency for Icinga 2 for the daemon itself. For the web part on the other end, it's definitely a must, as Icinga Web won't run without it (or at least the login would fail).